### DISCUSSION TITLE
Can copyq be set to only copy items w/ CTRL-C and not everytime anything is highlighted?

### DISCUSSION THREAD

**Discussion by Richie87G** [hluk/CopyQ#2519] [Q&A]
Can copyq be set to only copy items w/ CTRL-C and not every time anything is highlighted?

Copyq immediately stores any highlighted text.  Can it be set to only store items upon CTRL-C and not just by highlighting the text that I intend to replace?

Thanks all

**Comment by hluk** [comment]
Disable the "Store text selected using mouse" option in Preferences.
